.gitignore

```diff
@@ -22,6 +22,7 @@ public_html/wp-content/cache/
 public_html/wp-content/languages/
 public_html/wp-content/mu-plugins/blocks/build
 public_html/wp-content/mu-plugins/includes/slack/
+public_html/wp-content/mu-plugins/vendor/
```

Comment:
What gets installed here? Just the non-WP dependencies like the oAuth client? I'd probably want to keep plugins/themes in their respective `wp-content` folders, but I'm fine w/ libraries and stuff going here.

Reply:
Yeah, it's only libraries. Plugins and themes go where they're supposed to via installer scripts for `wordpress-plugin` and `wordpress-theme` package types.